docs/v1.0/plugin-management.txt

```diff
@@ -100,3 +100,43 @@ Fluentd will only load listed gems separated from shared gems, and will also pre
 
 In addition, if you update Fluentd's Ruby version, Bundler will re-install the listed gems for the new Ruby version.
 This allows you to avoid the C extension API compatibility problem.
+
+### For td-agent
+
+We can manage Fluentd and its plugins based on Gemfile with td-agent.
+
+Use following drop-in file /etc/systemd/system/td-agent.service.d/override.conf for td-agent 3.1.1:
+
+    :::text
+    [Service]
+    Environment='TD_AGENT_OPTIONS=--gemfile=/etc/td-agent/Gemfile --gem-path=/var/lib/td-agent/vendor/bundle'
+    ExecStart=
```

Comment:
This line is not needed?

Reply:
Yes.
See https://wiki.archlinux.org/index.php/Systemd#Examples